lib/tasks/rtd_location_create.rake

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+
+namespace :import do
+   desc "Add RTD Locations to Database"
+   task rtd: :environment do
+
+     location_data = RtdLocationsService.rtd_locations
+
+     creator= RtdLocationCreator.new
+     creator.create_record(location_data.body)
+
+     p "Import Successful!"
+   end
+ end
```

Comment:
Actually, I'd love to walk through this whole file, since I'm not familiar with rake